operator/pkg/ciliumendpointbatch/reconciler_test.go

```diff
@@ -0,0 +1,217 @@
+// Copyright 2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !privileged_tests
```

Comment:
out of curiosity, what's the point of `go:build` given the next line has `+build`?

Reply:
it's being added in golang 1.17

Reply:
Sorry missed your previous request, this was added by build infra.